maintainers.txt

```diff
@@ -0,0 +1 @@
+NuclearRedeye
```

Comment:
Just out of curiosity, what is the maintainers.txt for?

Reply:
It's our current standard for identifying the person responsible for maintaining the repository. Jenkins has some logic (for projects with pipelines) that it uses to send notifications. We're working through all our repo's assigning ownership (or archiving them where they aren't used anymore), but once done I'll be looking to migrate over to using CODEOWNERS files instead for the projects I own